src/libraries/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -312,6 +312,21 @@ public void ToString_NullFrame_ThrowsNullReferenceException()
             Assert.Equal(Environment.NewLine, stackTrace.ToString());
         }
 
+        [Fact]
+        public unsafe void ToString_FunctionPointerSignature()
+        {
+            // This is sepate from ToString_Invoke_ReturnsExpected since unsafe cannot be used for iterators
+            var stackTrace = FunctionPointerParameter(null);
+            if (PlatformDetection.IsMonoRuntime)
+            {
+                Assert.Contains("System.Diagnostics.Tests.StackTraceTests.FunctionPointerParameter(MonoFNPtrFakeClass x)", stackTrace.ToString());
```

Comment:
The Mono behavior is a bug that we'll want to fix eventually. Could you instead add `[ActiveIssue("https://github.com/dotnet/runtime/issues/11354", TestRuntimes.Mono)]` to this test?

(It doesn't make sense to cut a new bug for it; we have plenty of Mono bugs - Mono will either be fixed when 11354 is fixed, or there will be a new Mono-specific bug that we'll need to open as part of fixing 11354 to block all the failing tests added at that point.)